app/javascript/.storybook/main.js

```diff
@@ -8,8 +8,8 @@ module.exports = {
     '@storybook/addon-knobs',
     '@storybook/addon-actions',
     '@storybook/addon-links',
-    '@storybook/addon-a11y/register',
-    '@storybook/addon-notes/register-panel',
+    '@storybook/addon-a11y',
+    '@storybook/addon-docs',
```

Comment:
We can look at switching to `@storybook/addon-docs` in another PR. The docs add on preserves our existing documentation from notes, but we may need to do some work on the rendered components as it includes decorators like the theme selector in it.

Let's do the switch to addon-docs this for another PR.

Before with addon-notes

![image](https://user-images.githubusercontent.com/833231/90139905-fa98f980-dd46-11ea-8690-e0a606b7c511.png)

After with addon-docs

![image](https://user-images.githubusercontent.com/833231/90140038-2916d480-dd47-11ea-8818-dad59928b2bc.png)